CONTRIBUTING.md

```diff
@@ -1,21 +1,24 @@
 # How to Contribute
 
-We want your help to make Project Hydra great.
-There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
+We want your help to make the Samvera community great. There are a few guidelines
+that we need contributors to follow so that we can have a chance of
+keeping on top of things.
 
 ## Code of Conduct
 
-The Hydra community is dedicated to providing a welcoming and positive experience for all its
-members, whether they are at a formal gathering, in a social setting, or taking part in activities
-online.  Please see our [Code of Conduct](https://wiki.duraspace.org/display/hydra/Code+of+Conduct)
-for more information.
+The Samvera Community is dedicated to providing a welcoming and positive
+experience for all its members, whether they are at a formal gathering, in
+a social setting, or taking part in activities online. Please see our
+[Code of Conduct](CODE_OF_CONDUCT.md) for more information.
```

Comment:
I'd vote we link to a single code of conduct on the WIKI or in a master repo somewhere (samvera.github.io?).  That way we don't have to update N project repos when we want to update the code of conduct.  Just a thought.

Reply:
I'll fix this PR since I'm fixing all the others, but we'll talk about that in the CCMWG for sure.